### PR TITLE
Feat/v1

### PR DESCRIPTION
**Pull request by ossiggy**
- Adds all v2 methods, along with tests
 - Updates Readme
 - Bumps to v1.0.0

### CODE REVIEW

**Comment by radoshi** [comment]
Hi @ossiggy what's the status of committing this into the repo? There are some things here that we'd like to play around with (eg. CreditScore) that are not available in the released version of the lib. Thanks!

File: method/resources/Accounts/ExternalTypes.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    'financial_institution',
+    'payment_app',
+    'marketplace',
+    'payment_terminal',
+    'income_source'
+]
+
+
+class PlaidBalance(TypedDict):
+  available: Optional[int]
```
Comment (dwilliams27) [inline method/resources/Accounts/ExternalTypes.py:47]: Minor, but the spacing seems off on some of the classes in this file

File: method/resources/Accounts/Types.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    available_credit: Optional[int]
+    credit_limit: Optional[int]
+    interest_rate_percentage_max: Optional[float]
+    interest_rate_percentage_min: Optional[float]
+    interest_rate_type: Optional[AccountInterestRateTypesLiterals]
+    sub_type: Optional[AccountLiabilityCreditCardSubTypesLiterals]
+    usage_pattern: Optional[AccountLiabilityCreditCardUsageTypesLiterals]
+
+
+class AccountLiabilityCollection(AccountLiabilityBase):
```
Comment (dwilliams27) [inline method/resources/Accounts/Types.py:160]: Is this used anywhere?

**Review by dwilliams27 — APPROVED** [review]
Overall lgtm

File: method/resources/Accounts/ExternalTypes.py
Comment (sbilalh) [inline method/resources/Accounts/ExternalTypes.py:1]: could use some linting but otherwise fine

**Review by sbilalh — APPROVED** [review]
lgtm

File: method/resources/Accounts/ExternalTypes.py
Comment (sbilalh) [inline method/resources/Accounts/ExternalTypes.py:1]: could use some linting but otherwise fine

**Comment by ossiggy** [comment]
> Hi @ossiggy what's the status of committing this into the repo? There are some things here that we'd like to play around with (eg. CreditScore) that are not available in the released version of the lib. Thanks!

@radoshi this has been merged

**Pull request merged**